lib/mdx.tsx

```
@@ -0,0 +1,47 @@
+  Alert,
+  CodeTabs,
+  CodeTab,
+  DynamicFeaturesTable,
+  DynamicCommandsTable,
+  Inline,
+  code: (props: React.ComponentProps<'code'>) => {
+    let { children, ...rest } = props;
+    let content = typeof children === "string" ? children : String(children);
+    content = content.replace(/^`+|`+$/g, "");
+    if (!content.includes("\n")) {
+      return <Inline children={content} {...rest} />;
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)